app/api/v2/handlers/fact_source_api.py

```diff
@@ -22,44 +22,77 @@ def add_routes(self, app: web.Application):
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'id',
+                              'description': 'The id of the fact source',
+                              'schema': {'type': 'string'},
+                              'required': 'true'
+                            }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
-    @aiohttp_apispec.response_schema(SourceSchema(partial=True))
+    @aiohttp_apispec.response_schema(SourceSchema(partial=True), description='Returns a single Source dumped in SourceSchema format.')
     async def get_fact_source_by_id(self, request: web.Request):
         source = await self.get_object(request)
         return web.json_response(source)
 
+    @aiohttp_apispec.docs(tags=['sources'],
+                              summary='Create a fact source.',
+                              description='Returns a new fact source, given a name.',
+                              parameters=[{
+                                  'in': 'path',
+                                  'name': 'name',
+                                  'description': 'The name of the new fact source',
+                                  'schema': {'type': 'string'},
+                                  'required': 'true'
+                                }])
     @aiohttp_apispec.docs(tags=['sources'])
     @aiohttp_apispec.request_schema(SourceSchema)
-    @aiohttp_apispec.response_schema(SourceSchema)
+    @aiohttp_apispec.response_schema(SourceSchema, description='Returns a single Source dumped in SourceSchema format.')
     async def create_fact_source(self, request: web.Request):
         source = await self.create_on_disk_object(request)
         return web.json_response(source.display)
 
-    @aiohttp_apispec.docs(tags=['sources'], summary='All fields can be updated excluding "id" '
-                                                    'and "adjustments".')
+    @aiohttp_apispec.docs(tags=['sources'],
+                          summary='Update an existing fact source.',
+                          description='Returns an updated fact source. All fields in a fact source can be updated, except for "id" and "adjustments".')
     @aiohttp_apispec.request_schema(SourceSchema(partial=True))
-    @aiohttp_apispec.response_schema(SourceSchema)
+    @aiohttp_apispec.response_schema(SourceSchema, description='Returns a single Source dumped in SourceSchema format.')
     async def update_fact_source(self, request: web.Request):
         source = await self.update_on_disk_object(request)
         return web.json_response(source.display)
 
-    @aiohttp_apispec.docs(tags=['sources'])
+    @aiohttp_apispec.docs(tags=['sources'],
+                          summary='Update an existing or create a new fact source.',
+                          description='Returns an updated fact source. All fields in a fact source can be updated, except for "id" and "adjustments".')
```

Comment:
The PUT request also has an url parameter ID, so you can add the `parameters` section to the docs here as well